### PR TITLE
Add note about `astro deploment connections copy` not copying passwords

### PR DESCRIPTION
**Pull request by fritz-astronomer**

### CODE REVIEW

**Comment by fritz-astronomer** [comment]
forgot about this @astronomer/docs give me a quick review? I don't think anything has changed since I wrote it

File: astro/cli/astro-deployment-connection-copy.md [hunk 1]
```diff
@@ -13,6 +13,12 @@ This command is only available on Astro.
 
 :::
 
+:::alert
+
+This command copies ALL connections from one instance to another, but does not copy encrypted data such as passwords or some `extras` such as when it contains private keys.
```
Comment (lzdanski) [inline astro/cli/astro-deployment-connection-copy.md:18]:
```suggestion
This command copies **all** connections from one instance to another, but does not copy encrypted data, like passwords, or some `extras`, such as when they contain private keys.
```

**Review by lzdanski — COMMENTED** [review]
small copy recommendations!

File: astro/cli/astro-deployment-connection-copy.md [hunk 2]
```diff
@@ -13,6 +13,12 @@ This command is only available on Astro.
 
 :::
 
+:::alert
```
Comment (lzdanski) [inline astro/cli/astro-deployment-connection-copy.md:16]:
```suggestion
:::warning
```

**Review by lzdanski — APPROVED** [review]

**Pull request merged**